core/src/test/scala/unit/kafka/server/AbstractApiVersionsRequestTest.scala

```diff
@@ -59,15 +68,40 @@ abstract class AbstractApiVersionsRequestTest(cluster: ClusterInstance) {
     } finally socket.close()
   }
 
-  def validateApiVersionsResponse(apiVersionsResponse: ApiVersionsResponse): Unit = {
-    val expectedApis = ApiKeys.zkBrokerApis()
+  def validateApiVersionsResponse(apiVersionsResponse: ApiVersionsResponse, controllerApi: Boolean = false): Unit = {
```

Comment:
Instead of passing through `controllerApi` as a flag, maybe we could pass the listener?

Reply:
Done.